routes/rooms.go

```diff
@@ -56,6 +55,32 @@ func GetEmergencyRoomById(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, resp)
 }
 
+// GetEmergencyRooms godoc
+// @Summary    get all emergency rooms
+// @Tags       emergency-rooms
+// @Produce    json
+// @Param      page                            query       uint                    false   "0-indexed page number, 0 is assumed when omitted"
+// @Param      page_size                       query       uint					   false   "page size, 100 is assumed when omitted"
+// @Success    200                             {object}    []uuid.UUID
```

Comment:
My mistake from earlier. This should still be a `GetMultipleERsResponse`. But `GetMultipleERsResponse` should contain an array of uuids instead of GetSingleERResponse. (line 229)

Without this we actually don't return any pagination information.